include/deal.II/base/vectorization.h

```diff
@@ -925,7 +925,7 @@ vectorized_transpose_and_store(const bool                            add_into,
 // for safety, also check that __AVX512F__ is defined in case the user manually
 // set some conflicting compile flags which prevent compilation
 
-#  if DEAL_II_COMPILER_VECTORIZATION_LEVEL >= 3 && defined(__AVX512F__)
+#  if DEAL_II_VECTORIZATION_WIDTH_IN_BITS >= 512 && defined(__AVX512F__)
```

Comment:
should we remove the ``define`` here and fail instead?

Reply:
Again, it is to hedge for the non-x86-64 case.